### DISCUSSION TITLE
Historical data older than 2023

### DISCUSSION THREAD

**Discussion by martincivan** [br-g/openf1#33] [General]
Hello,

first of all thank you for this awesome project!
I have noticed that few weeks ago all older data have disappeared now only data from 2023 forward are available. 
Is there any chance to get some older data? 

Thanks for any help!

Martin

**Comment by br-g** [comment]
Hello Martin,

Thanks for your kind words!

Data prior to 2023 has never been added to the database so far, but I plan to do it soon.
I first need to finish implementing some optimizations that will significantly reduce the cost of inserting new data.

In the meantime, if you really need this historical data, I recommend that you use [FastF1](https://docs.fastf1.dev).
